### PR TITLE
Updated dependencies.

### PR DESCRIPTION
**Pull request by jeffijoe**
Tests have not been run, but it works on my end.

### CODE REVIEW

**Comment by jeffijoe** [comment]
Well apparently everything is OK. Thanks Travis!

**Comment by jmeas** [comment]
The deps in package.json and bower.json should be the same. I know they're not, now...but...that's my mistake :see_no_evil: lol

Do the tests not pass if Underscore isn't >= 1.8.3? I'd like to keep the deps as flexible as possible.

**Comment by jeffijoe** [comment]
Don't know. :) 

~ Jeff Hansen

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.intercept",
-  "version": "0.3.3",
+  "version": "0.3.4",
```
Comment (thejameskyle) [inline package.json]: This should probably not be part of the PR.

Comment (jeffijoe) [inline package.json]: Sorry, I don't do this often.

~ Jeff Hansen

**Comment by jmeas** [comment]
No prob, @jeffijoe ! Usually I'll bump the version / write the CHANGELOG right before a release. So the PR you write can just focus on a feature.

Chances are we just need to increase the maximum version for Underscore/Backbone without changing the current minimum. I can do that tonight if you don't have time.

**Comment by jeffijoe** [comment]
Go ahead :) 

~ Jeff Hansen

**Comment by jmeas** [comment]
Resolved in #30 and #31
